tests/unit_tests.py

```diff
@@ -298,6 +298,35 @@ def test_nlp_body(self):
         self.assertCountEqual(KEYWORDS, self.article.keywords)
 
 
+class ContentExtractorTestCase(unittest.TestCase):
+    """Test specific element extraction cases"""
+
+    def setUp(self):
+        self.extractor = newspaper.extractors.ContentExtractor(Configuration())
+        self.parser = newspaper.parsers.Parser
+
+    def _get_title(self, html):
+        doc = self.parser.fromstring(html)
+        return self.extractor.get_title(doc)
+
+    def test_get_title_basic(self):
+        html = '<title>Test title</title>'
+        assert self._get_title(html) == 'Test title'
```

Comment:
Wrong kind of assertion, should be `self.assertEqual('Test title', self._get_title(html))`